code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -78,6 +79,47 @@
 		return FALSE
 	return ..()
 
+/mob/living/simple_animal/hostile/abnormality/mountain/Moved()
+	. = ..()
+	var/turf/floor = get_turf(src)
+	for(var/obj/thing in floor)
+		if(istype(thing, /obj/effect/decal/cleanable/blood))
```

Comment:
I don't think it should heal from blood. Also revert blood_volume changes to keep the blood effects on damage.

Reply:
Why _not_? The thing is already easy to suppress at low levels, blood is STUPIDLY easy to clean, and I can just make him NOT bleed. It's also 0.005% to 0.5% healing entirely depending on the QUANTITY of blood. It's just more incentive and a different kind of incentive to keep your fucking facility clean when he's around.

Reply:
That's simple: Blood is everywhere and comes from everyone. MoSB should only gain stages from humans and what was once human, so, organs and bodyparts.